pkg/engine/planner/logical/logical_test.go

```diff
@@ -0,0 +1,58 @@
+package logical
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPlan_String(t *testing.T) {
+	// Build a simple query plan:
+	// SELECT id, name FROM users WHERE age > 21 SORT BY age ASC
```

Comment:
The plan below does not really represent this SQL statement.
Maybe you wanna use a LogQL statement that can be represented by the current instructions instead?

Reply:
Good idea, I'll update all the tests to move away from the SQL query examples